packages/editor-ui/src/constants.ts

```diff
@@ -458,6 +458,13 @@ export const enum VIEWS {
 	WORKER_VIEW = 'WorkerView',
 }
 
+export const EDITABLE_VIEWS = [
+	VIEWS.WORKFLOW,
+	VIEWS.NEW_WORKFLOW,
+	VIEWS.LOG_STREAMING_SETTINGS,
```

Comment:
It's weird that log streaming is here. I am not seeing it touched in this PR. Could we safely remove it?

Reply:
I copied over the previous nonReadOnly(editable) views. I guess Log streaming settings route has some expression fields when setting up connection so it's a way to make them editable
![CleanShot 2024-01-04 at 15 15 16](https://github.com/n8n-io/n8n/assets/12657221/e3aa4f88-a64f-462c-8bbf-7b5872761e7f)

Reply:
cannot see expressions being very useful there. But that's probably why. If we fix that bug, then maybe we can remove it then.